go-apps/meep-tc-sidecar/main.go

```diff
@@ -249,21 +254,40 @@ func initMeepSidecar() (err error) {
 	// Connect to Metric Store
 	metricStore, err = met.NewMetricStore(scenarioName, sandboxName, influxDBAddr, redisAddr)
 	if err != nil {
-		log.Error("Failed connection to Redis: ", err)
+		log.Error("Failed connection to Redis. Error: ", err)
+		return err
+	}
+
+	// Create & initialize pinger instance
+	pinger, err = New(opts.bind4, opts.bind6)
+	if err != nil {
+		log.Error("Failed to create Pinger. Error: ", err)
+		return err
+	}
+	if pinger.PayloadSize() != uint16(opts.payloadSize) {
```

Comment:
Why check `pinger.PayloadSize() != uint16(opts.payloadSize)` - just set it.

Reply:
Modified as suggested.